qcengine/tests/test_procedures.py

```diff
@@ -48,7 +48,7 @@ def test_geometric_psi4(input_data, optimizer):
     # Check keywords passing
     for single in ret.trajectory:
         assert "scf_properties" in single.keywords
-        assert "WIBERG_LOWDIN_INDICES" in single.extras["qcvars"]
+        assert "WIBERG_LOWDIN_INDICES" in single.extras["qcvars"] or "WIBERG LOWDIN INDICES" in single.extras["qcvars"]
```

Comment:
Oh no, is either form possible right now?

Reply:
The underscored version has been deprecated since v1.4, and Ben has been pushing to not store two copies. The opt-disp env is hitting the space version, so I figured both would cover all bases. Do you foresee a problem?

Reply:
No problem, that's enough for me. May want to add in a comment over this line with this info, and perhaps a `TODO` to eventually remove.